### DISCUSSION TITLE
文科关于末流211与普通本科的选择

### DISCUSSION THREAD

**Discussion by Jeremy050209** [CollegesChat/university-information#169] [Q&A]
河北文科考生557 能够到像石河子新疆大学一类的末流211的差专业（还有一个什么贵州大学中外合办的旅游管理） 但是我想学法学或者汉语言 想去重庆成都什么的活跃一点的城市 但是这样的话就只能上个普通本科院校 以后准备考研 请问该怎么选择比较好

**Comment by gggratitude** [comment]
同年考生，个人建议选个好二本仔细挑一下有可能升一本的大学，瞄准汉语言或者法学，祝你前程似锦未来可期

**Comment by HustreLiny** [comment]
文科建议多考虑城市，其次是专业，石河子这种211含金量还不如好点的普通本科
